src/dropbox_client.rs

```diff
@@ -1,36 +1,59 @@
-use std::env;
+use serde_json::json;
 use reqwest::{Client, ClientBuilder};
 use reqwest::header::{HeaderMap, AUTHORIZATION, CONTENT_TYPE};
 use crate::domain::{ItemList};
 
-fn make_client() -> Client {
-    let dropbox_token = env::var("DROPBOX_TOKEN").expect("You have to provide a Dropbox API token");
-    let authorization_header = format!("Bearer {}", dropbox_token);
+const DROPBOX_FILE_PATH: &'static str = "/pudeuko/data.json";
+const DROPBOX_DOWNLOAD_URL: &'static str = "https://content.dropboxapi.com/2/files/download";
+const DROPBOX_UPLOAD_URL: &'static str = "https://content.dropboxapi.com/2/files/upload";
 
-    let mut headers = HeaderMap::new();
-    headers.insert(AUTHORIZATION, authorization_header.parse().unwrap());
-
-    let client = ClientBuilder::new().default_headers(headers).build().unwrap();
-    client
+pub struct DropboxClient {
+    client: Client,
+    download_headers: HeaderMap,
+    upload_headers: HeaderMap,
```

Comment:
Jest sens trzymać to na poziomie obiektu? Tam chociażby wymuszasz sposób zapisu na dropbox co jest jawnie zależne od metody, trzymałbym tylko składowe nagłówków jak token a nie całą hashmapę